taylorjohnson141/travel-tracker-tj
# Add Sass

What’s this PR do?
This PR adds the final sass to the page 
Where should the reviewer start?
src/base.scss
How should this be manually tested?

run webpack and see color changes, run lighthouse

Any background context you want to provide?
Needed to add Sass
What are the relevant tickets?

Screenshots (if appropriate)

Questions:
